src/main/resources/de/tum/cit/ase/ares/api/configuration/essentialFiles/java/EssentialClasses.yaml

```
@@ -0,0 +1,30 @@
+essentialJavaClasses: []
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)